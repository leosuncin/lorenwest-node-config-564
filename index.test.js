test("Default config", function() {
  expect(require(".")).toMatchSnapshot();
});
