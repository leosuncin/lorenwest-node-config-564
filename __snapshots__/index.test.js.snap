// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default config 1`] = `
Config {
  "app": Object {
    "port": 1337,
    "secret": "secret",
  },
  "cors": Object {
    "credentials": true,
    "origin": "http://localhost:1337",
  },
}
`;
